schema/fmi3Variable.xsd

```diff
@@ -381,6 +381,7 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 		<xs:attribute name="canHandleMultipleSetPerTimeInstant" type="xs:boolean"/>
 		<xs:attribute name="declaredType" type="xs:normalizedString"/>
 		<xs:attribute name="clockReference" type="xs:unsignedInt" use="optional"/>
+		<xs:attribute name="clockElementIndex" type="xs:unsignedLong" use="optional"/>
 		<xs:attribute name="intermediateAccess" type="xs:boolean" use="optional"/>
 	</xs:complexType>
 
```

Comment:
An additional change in fmi3Variable.xsd is necessary: The fmi3Clock's start attribute 
`<xs:attribute name="start" type="xs:boolean"/>`
must be a list (same as for other variables):
```
<xs:attribute name="start">
	<xs:simpleType>
		<xs:list>
			<xs:simpleType>
				<xs:restriction base="xs:boolean"/>
			</xs:simpleType>
		</xs:list>
	</xs:simpleType>
</xs:attribute>
```

Reply:
Yes indeed, will be added.